### ISSUE TITLE
How does prompt.yaml work

### ISSUE THREAD

**Issue by sauradip** [rohitgandikota/sliders#65]
In prompt.yaml i found this following file 

https://github.com/rohitgandikota/sliders/blob/4560f5f01241e0ba3c289d1e4e1ba1bf1a7c927f/trainscripts/textsliders/data/prompts.yaml#L1

and 

https://github.com/rohitgandikota/sliders/blob/4560f5f01241e0ba3c289d1e4e1ba1bf1a7c927f/trainscripts/textsliders/data/prompts.yaml#L10

So when we do this, does it mean we are trying to create a separate disentangled slider for both male and female ? 

And also did you try to learn colors/shapes parameters ? like (bright<->dark) and (small<->large ) ?

**Comment by rohitgandikota** [comment]
Yes, when we do that - we are essentially introducing a 2d dimension of disentanglement. You can also include it in along with the actual `--attributes` argument, but this is a more methodological way to do it. 

We did there is a night time slider which is present in the pretrained sliders list. We did not try any shape related slider, that would be a cool slider. Feel free to share if you built any cool sliders.

**Comment by rohitgandikota** [comment]
I am trying to understand why you are targeting a special token. Because the slider itself acts like a special token. You can leave the target empty `""` or use a specific concept whose shape you want to alter. Since you are using "large object" maybe you can try target "object" 

Essentially, if you want to change the shape use slider scale !=0 and whenever you do not want the shape change keep the scale=0 during inference (hence slider acts like that proxy special token)

**Comment by rohitgandikota** [comment]
Ahh, i see what you are doing. That's clever! 

Although we have not tested for such cases. In the beginning it sounded like this question can be solved using some customization techniques like dreambooth, but the question might be more complex as I try to understand.

So here's my suggestion:
1. Try to alter the shape of just watermelon first. By that I mean use target as "watermelon" positive as "watermelon, large" and unconditional as "watermelon, small". And try to get this working. Some ways to improve this is by setting large `start_noise` parameter in inference (maybe 900). If that does not work then you can increase iterations in training (also maybe increase learning rate)

2. If this works well, then try to use the same parameters and train the `*` setting. 

Let me know if any of them works for you.

**Comment by rohitgandikota** [comment]
Closing the issue - please free to reopen if the issue persists
